### PR TITLE
Adding example JSON files for triggers and props

### PR DESCRIPTION
**Pull request by dgmccart**

### CODE REVIEW

**Comment by aliddell** [comment]
These are the same JSON files from #8 and #9, right? There's got to be a way to include them as snippets or something directly, so you don't have to change them in two different places if anything changes.

**Comment by dgmccart** [comment]
I agree. Let's discuss at the sync today

**Comment by melissawm** [comment]
We have changed the repo since this PR was created to include a separate folder for tutorials. Is it ok if I merge the latest changes into your PR @dgmccart ? I can then make a suggestion about how to include those files using [mkdocs-include-markdown-plugin](https://github.com/mondeja/mkdocs-include-markdown-plugin)

**Comment by melissawm** [comment]
This should work :smile:

File: docs/tutorials/props_json.md [hunk 1]
```diff
@@ -49,208 +49,9 @@ with open("sample_props.json", "w") as outfile:
...
-                },
-                "write_delay_ms": 0.0
-            }
-        }
-    ]
-}
-```
+~~~json
+{% include "./sample_props.json" %}
+~~~
```
Comment (aliddell) [inline docs/tutorials/props_json.md:54]: 🎉🎉🎉

**Comment by dgmccart** [comment]
Looks like this is ready to merge @aliddell or @melissawm

**Review by aliddell — APPROVED** [review]

**Pull request merged**